backend/src/Edna/Setup.hs

```diff
@@ -1,28 +1,41 @@
 module Edna.Setup
-  ( EdnaContext (..)
-  , Edna
-  , edConfig
+  ( Edna
   , runEdna
+
+  , EdnaContext (..)
+  , edConfig
+  , edConnectionPool
   ) where
 
 import Universum
 
 import Lens.Micro.Platform (makeLenses)
+import RIO (RIO, runRIO)
+import System.IO (hSetEncoding, utf8)
 
 import Edna.Config.Definition (EdnaConfig(..))
-import RIO (RIO, runRIO)
+import Edna.DB.Connection (ConnPool(..), withPostgresConn)
 
 data EdnaContext = EdnaContext
   { _edConfig :: !EdnaConfig
+  , _edConnectionPool :: !ConnPool
   }
 
 makeLenses ''EdnaContext
 
 type Edna = RIO EdnaContext
 
+setEncoding :: IO ()
+setEncoding = do
```

Comment:
There is `with-utf8` library maintained by us, let's use it. Just wrap `main` into `withUtf8`.